server/models.py

```diff
@@ -31,17 +45,23 @@ class Activity(models.Model):
     duration = models.DurationField()
     date = models.DateField()
```

Comment:
Maybe we should use a `DateTimeField()` here like you used in the `Comment` class?

Reply:
3f0922ad3dcb40c02f1d862fecf0a61d45bc43f3